components/registration/RegisterForm.vue

```diff
@@ -167,7 +167,7 @@ export default class RegisterForm extends mixins(BaseMixin, ValidationMixin) {
         try {
             this.$v.$touch()
             if (this.$v.$invalid || !this.isPasswordValid) {
-                return
+                // empty
```

Comment:
is this correct ?

Reply:
previously build was failing due to having an empty return

`// empty `as suggested by eslint, seemed to solve it

Reply:
You would still need an return to break out of the conditions

Reply:
fixed